posix.mak

```diff
@@ -150,7 +150,7 @@ ifdef ENABLE_COVERAGE
 override DFLAGS  += -cov
 endif
 
-UDFLAGS=-unittest
+UDFLAGS=-unittest -version=StdUnittest -version=CoreUnittest
```

Comment:
Why include druntime's unittests with `CoreUnittest`?

Reply:
Phobos uses https://github.com/dlang/druntime/blob/9baf9735c923a9bbb72b8e4ee05f513ea0499bbc/src/core/time.d#L120-L123 to run its unittests. But that `version (unittest)` will change into `version (CoreUnittest)` (https://github.com/dlang/druntime/pull/2902), so the flag has to be there.

Reply:
Right, -`unittest` cascades into druntime. Thanks for the explaination.

Reply:
Should now be not necessary with my changes to https://github.com/dlang/druntime/pull/2902.

Reply:
Addressed in https://github.com/dlang/phobos/pull/7356.